_locales/en/messages.json

```diff
@@ -734,6 +734,9 @@
     "message": "Live reload",
     "description": "The label of live-reload feature"
   },
+  "manageExecutionOrder": {
+    "message": "Change execution order"
```

Comment:
`Reorder` for the button label should be fine, @narcolepticinsomniac, WDYT?

Maybe also add a title to the minipanel: `Style injection order`